README.md

```diff
@@ -0,0 +1,196 @@
+![Stream Cli](./assets/logo.png)
+
+# Stream CLI
+
+---
+> ## 🚨 **Breaking changes in v1.0 <**
+> We have completely rewritten the Node.JS CLI to Go in 2022 Q1. Some of the changes:
+> - The installation process is easier since it doesn't have any prerequisites (such as NPM). You can just simply download the executable and run it.
+> - The name of the executable is `stream-cli` instead of `stream` to avoid conflicts with an existing tool ([imagemagick](https://github.com/GetStream/stream-cli/issues/33)). But you can rename it if you want to.
+> - The command invocation is `stream-cli chat [verb-noun] [args] [options]` instead of `stream [verb:noun] [args] [options]`. The most obvious change is using dash instead of colon. We also added the `chat` keyword to preserve domain for our other product [Feeds](https://getstream.io/activity-feeds/).
+> - The initial version has a bit smaller feature set, but we'll bring it up to par by 2022 Q2.
```

Comment:
i had to read it twice, `initial version` i thought you were refering to the node version. can we make it clearer?

Reply:
rephrased ✅